mlflow/tensorflow.py

```diff
@@ -994,7 +994,7 @@ def fit(self, *args, **kwargs):
                 early_stop_callback = _early_stop_check(tmp_list[5])
                 tmp_list[5], log_dir = _setup_callbacks(tmp_list[5], log_models)
                 args = tuple(tmp_list)
-            elif "callbacks" in kwargs:
+            elif kwargs.get("callbacks"):
```

Comment:
`kwargs["callbacks"]` may still be `None`, as this is the default value for the argument. For example, `fit_generator()` calls `fit()` with `callbacks=None` in TensorFlow >= 2.1.0. Many of our other autologging integrations address this case, but this one does not.